ddtrace/settings/config.py

```diff
@@ -156,6 +156,9 @@ def _parse_propagation_styles(styles_str):
             log.warning("Unknown DD_TRACE_PROPAGATION_STYLE: {!r}, allowed values are %r", style, PROPAGATION_STYLE_ALL)
             continue
         styles.append(style)
+    # Remove "none" if it's present since it lacks a propagator
+    if _PROPAGATION_STYLE_NONE in styles:
```

Comment:
nit: Can we prevent "none" from being added to styles (update `if not style:` to `if not style or style == "none"`). This is slightly more performant.